.chronus/changes/change-response-to-be-array-2024-8-14-11-27-13.md

```diff
@@ -0,0 +1,9 @@
+---
+changeKind: breaking
+packages:
+  - "@azure-tools/typespec-client-generator-core"
+---
+
+1. The type of `responses` and `exceptions` in `SdkHttpOperation` changed from `Map` to array.
+2. The type of `responses` in `SdkHttpOperationExample` changed from `Map` to array.
+3. `SdkHttpResponse` adds a new property `statusCodes` to store its corresponding status code or status code range.
```

Comment:
nit: i don't know if chronus could handle markdown.

Reply:
this works as pure text as well I believe

Reply:
but since the file is in `md`, it makes no sense if it cannot handle it properly - therefore I believe it should be able to handle it